src/Domain/Enum/MethodType.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the eluceo/iCal package.
+ *
+ * (c) 2021 Markus Poerschke <markus@poerschke.nrw>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Eluceo\iCal\Domain\Enum;
+
+final class MethodType
```

Comment:
```suggestion
final class Method
```

The suffix "type" is not needed for this Enum.

Reply:
Sure, i'm updating.